spec/controllers/v1.0/actions_controller_spec.rb

```diff
@@ -59,4 +58,40 @@
       end
     end
   end
+
+  describe 'POST /requests/:request_id/actions' do
```

Comment:
You've added the spec in `actions_controller_spec` indicating it belongs to actions controller.